infrastructure/identity-aware-proxy/main.py

```diff
@@ -0,0 +1,7 @@
+import sys
+from make_iap_request import *
+
+if __name__ == "__main__":
+	# sys.argv[1] = URL
+	# sys.argv[2] = IAP Client Id
+	print make_iap_request(sys.argv[1],sys.argv[2])
```

Comment:
Using sys.argv directly is relatively fragile; using argparse (https://docs.python.org/2/howto/argparse.html) makes your intentions more clear, and lets you include a --usage or --help flag

Reply:
Done